peachjam/js/components/DocumentContent/index.ts

```diff
@@ -178,15 +178,15 @@ class DocumentContent {
       this.tocController?.addEventListener('itemTitleClicked', (e) => {
         if (this.originalDocument && this.documentElement) {
           const id = (e as CustomEvent).detail.target.getAttribute('href');
-          if (id) {
+          if (!id || id === '#') {
+            this.documentElement.replaceChildren(...Array.from(this.originalDocument.children).map(node => node.cloneNode(true)));
```

Comment:
Minor: I don' think you need the `...` -- that makes a copy of the existing array AFAIK. You're already cloning the nodes.

Reply:
it breaks when we remove the ellipses,

Reply:
ugh, javascript